rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -178,43 +179,43 @@ def copyImage(self, array):
         b.rocalCopyToOutput(
             self._handle, np.ascontiguousarray(out, dtype=array.dtype))
 
-    def copyToTensor(self, array,  multiplier, offset, reverse_channels, tensor_format, tensor_dtype):
+    def toTensor(self, array,  multiplier, offset, reverse_channels, tensor_format, tensor_dtype):
```

Comment:
We still need to define this fn as copyToExternalTensor() or something similar since the api does the tensor conversion as well copying to external memory. Applies to all similar functions

Reply:
Renamed all the python API functions to copyToExternalTensor(). Still keeping the rocAL API functions as toTensor